### DISCUSSION TITLE
Kibana issue after update

### DISCUSSION THREAD

**Discussion by m9o7h** [Security-Onion-Solutions/securityonion#10635] [General] [answered]
Dears, 
I've updated my distributed SO system that have 2.3.91 version to the 2.3.250 version,
Every thing is working fine except for Kibana discover tab (https://XX.XX.XX.XX/kibana/app/discover)
I'm getting the following error 

t.at is not a function

`TypeError: t.at is not a function
    at https://XX.XX.XX.XX/kibana/61224/bundles/plugin/data/kibana/data.plugin.js:1:312992
    at Array.forEach (<anonymous>)
    at n (https://XX.XX.XX.XX/kibana/61224/bundles/plugin/data/kibana/data.plugin.js:1:312956)
    at s (https://XX.XX.XX.XX/kibana/61224/bundles/plugin/discover/kibana/discover.chunk.0.js:1:10138)
    at https://XX.XX.XX.XX/kibana/61224/bundles/plugin/discover/kibana/discover.chunk.0.js:1:10215
    at Array.map (<anonymous>)
    at a (https://XX.XX.XX.XX/kibana/61224/bundles/plugin/discover/kibana/discover.chunk.0.js:1:10207)
    at https://XX.XX.XX.XX/kibana/61224/bundles/plugin/discover/kibana/discover.chunk.0.js:1:8442
    at https://XX.XX.XX.XX/kibana/61224/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:334:27724
    at t.s._next (https://XX.XX.XX.XX/kibana/61224/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:6:33456)`

I've found similar issue with Elastic 8.7.0 on this [link ](https://discuss.elastic.co/t/elastic-8-7-0-error-in-discover-tab/332329/1) that fixed by updating the elastic

also find my [soup.log](https://github.com/Security-Onion-Solutions/securityonion/files/11829122/soup.log)

**Comment by dougburks** [comment, marked as answer]
Have you tried shift-reload in your browser as shown at https://docs.securityonion.net/en/2.3/soup.html#kibana?

Have you tried clearing your browser cache?

Have you tried a different browser?

**Reply by m9o7h** [reply]
Dear, I've tried another browser and it works.

thanks
